### ISSUE TITLE
[Bug] dicom-image-loader: Image all white - wrong min/max pixel values

### ISSUE THREAD

**Issue by muellnerlisa** [cornerstonejs/cornerstone3D#757]
### Describe the Bug

Image is displayed all white, possibly due to wrong min/max values. When checking the min/max pixel values there is something odd. 
In the dicom file itself the values are: 
- Smallest Image Pixel Value: 8192
- Largest Image Pixel Value: 24575

When debugging cornerstone the values of the image are: 
- minPixelValue: 26624
- maxPixelValue: 30613

Since the window center/window width values are 16384/16384, cornerstones min/max pixel values are clearly outside of the window. 

Our suspicion is that something in the decoding of the JPEG goes wrong. The transfer syntax is: 1.2.840.10008.1.2.4.70

We are using the latest dicom-image-loader version 1.11.3.

### Steps to Reproduce

Open this [example dicom file](https://github.com/cornerstonejs/cornerstone3D/files/12462469/too-bright.zip) using the dicom-image-loader and see how its initially displayed too bright.

### The current behavior

The image is displayed all white (see image), only after changing the window center/width to values around 29348/7417 the image seems somehow okay. 

![actual](https://github.com/cornerstonejs/cornerstone3D/assets/92728067/abd5c697-4b8f-4191-bafb-b9ecd276c169)

### The expected behavior

This JPEG rendering provided by our backend shows how the image would look like if rendered with correct intensity values.
![expected](https://github.com/cornerstonejs/cornerstone3D/assets/92728067/7536c2f4-b424-4183-8685-981decc85439)

### OS

Windows 10, Ubuntu 23.04

### Node version

v16.20.0, v18.17.1

### Browser

Chrome 116.0.5845.111, Firefox 116.0.3

**Comment by sedghi** [comment]
yeah seems too bright, i will investigate

**Comment by muellnerlisa** [comment]
@sedghi I just wanted to ask if there are any updates on this issue?

**Comment by sedghi** [comment]
Not yet

**Comment by jbocce** [comment]
I have done some investigation into this...

- I have confirmed what @muellnerlisa has reported...
   - using the dicom-image-loader the image appears completely white
   - the min and max pixel values via debugging respectively are 26624 and 30613
- loading the study from orthanc displays it correctly and the min and max pixel values are 8192 and 24575 respectively
- 3D slicer displays the study correctly and the min and max pixel values are as reported in the main dicom file

**Comment by sedghi** [comment]
Yeah based on my investigation also this is the JPEGLossless codec, @muellnerlisa can you let us know how have you encoded this image?

**Comment by muellnerlisa** [comment]
@sedghi @jbocce Thank you for investigating! Unfortunately, I don't have specific details on the exact encoding of the image. It is part of a large anonymized dataset sourced from multiple origins.

**Comment by daker** [comment]
@sedghi & jbocce i have proposed a fix

<img width="1188" height="1210" alt="Image" src="https://github.com/user-attachments/assets/8f6d2dbc-ac71-4319-b617-460d88724c2b" />
